frontend/app/templates/components/account-form.hbs

```diff
@@ -43,23 +40,23 @@
 
                   <div class="form-group col-sm-6">
                     <div class="row">
-                      <Input class="form-control " @value={{this.changeset.cleartextPassword}} @name="cleartextPassword" @type="password"/>
+                      <Input class="form-control mb-10" @value={{mut this.changeset.cleartextPassword}} @name="cleartextPassword" @type="password" @key-up={{action "updatePasswordScore"}}/>
+                      {{ember-progress-bar progress=this.passwordScore options=(hash strokeWidth=3 duration=1)}}
```

Comment:
Nice, I like the simple look. 
What about colors? Possible?
A subtle red, orange and green would be cool I think. But not sure if worth the effort...

Reply:
Possible. But you need to reload the ProgressBar everytime you want to change the Color.